reporting-analysis/load-test/src/reporting-analysis_get_flow_list.js

```diff
@@ -0,0 +1,122 @@
+import http from 'k6/http';
+import { check } from 'k6';
+import { generateFakeStringArray, randomString } from './modules/helpers.js';
+
+export let options = {
+	summaryTrendStats: ['avg', 'min', 'med', 'max', 'p(95)', 'p(99)', 'p(99.99)', 'count'],
+	stages: [
+		{ duration: '10s', target: 5 }, // simulate ramp-up of traffic from 1 to 50 users over 1 minutes.
```

Comment:
look at the comment made on previous test ( configuration mismatch vs comment )

Reply:
done